.github/workflows/main.yml

```diff
@@ -35,4 +35,4 @@ jobs:
                 platforms: linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x
                 push: true
                 tags: |
-                    zubairov/mqtt-exporter:latest
+                    zubairov/${{ secrets.DOCKERHUB_USERNAME }}:latest
```

Comment:
You did a little mistake:

`${{ secrets.DOCKERHUB_USERNAME }}/mqtt-exporter:latest`

Reply:
yeah, sorry 🤦